crates/typst-library/src/text/quote.rs

```diff
@@ -74,10 +79,10 @@ pub struct QuoteElem {
     /// ```example
     /// #quote(attribution: [René Descartes])[cogito, ergo sum] \
     ///
-    /// #show quote.where(block: false): it => [
-    ///   "#it.body"
-    ///   #if it.attribution != none [(#it.attribution)]
-    /// ]
+    /// #show quote.where(block: false): it => {
+    ///   ["] + h(0pt, weak: true) + it.body + h(0pt, weak: true) + ["]
+    ///   if it.attribution != none [ (#it.attribution)]
+    /// }
```

Comment:
Please add blank lines between show rules and following content (and generally between "blocks" of markup). I think there are some other instances of this.